facebook-commerce-events-tracker.php

```diff
@@ -1258,6 +1267,32 @@ public function get_tracked_events() {
 			return $this->tracked_events;
 		}
 
+		/**
+		 * Gets the pending events awaiting to be sent
+		 *
+		 * @return array
+		 */
+		public function get_pending_events() {
+			return $this->pending_events;
+		}
+
+		/**
+		 * Send pending events.
+		 */
+		public function send_pending_events() {
+
+			$pending_events = facebook_for_woocommerce()->get_integration()->events_tracker->get_pending_events();
+
+			if ( empty( $pending_events ) ) {
+				return;
+			}
+
+			foreach ( $pending_events as $event ) {
+
+				$this->send_api_event( $event);
```

Comment:
Space after `$event` is missing.
```suggestion
				$this->send_api_event( $event );
```